tests/test_feature_extraction_layoutlmv2.py

```diff
@@ -159,6 +159,15 @@ def test_call_pytorch(self):
 
         # Test not batched input
         encoded_images = feature_extractor(image_inputs[0], return_tensors="pt").pixel_values
+        print(encoded_images.shape)
+        print(
+            (
+                1,
+                self.feature_extract_tester.num_channels,
+                self.feature_extract_tester.size,
+                self.feature_extract_tester.size,
+            )
+        )
```

Comment:
These print statements should be removed

Reply:
Done 😂